react/week1/todolist/src/todos.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const TodoItems = (props) => {
+    return (
+        <h2>
+            {props.activity},{props.duration}
+        </h2>
+    )
+}
+
+const RenderedTodos = (props) => {
+    return (
+        <ol>
+            <h1 className="todo-header">Todo List</h1>
+          
+                {props.todos.map(todo => {
+                return  <li> <TodoItems activity={todo.activity} duration={todo.duration} key={todo.id} />
+                 </li>
```

Comment:
maybe work a little with the formatting here. for example:

    return (
      <li>
        <TodoItems
          activity={todo.activity}
          duration={todo.duration}
          key={todo.id}
        />
      </li>
    );

is a little easier to read.